constructs/check-group-v2.mdx

```diff
@@ -228,6 +228,43 @@ If **set to** `'global'`, it overrides the alert settings of all checks in the g
 
 </ResponseField>
 
+## Reference an existing group by ID
+
+To add checks to an existing group in your account, find the group ID in the Checkly web UI or via the API and reference it using `CheckGroupV2.fromId()`.
+
+<Tabs>
+  <Tab title="Web UI">
+    Navigate to the group in the Checkly UI and copy the group ID from the URL or the group details.
+  </Tab>
+  <Tab title="API">
+    ```bash
+    curl -H "Authorization: Bearer YOUR_API_KEY" \
+         -H "X-Checkly-Account: YOUR_ACCOUNT_ID" \
+         https://api.checklyhq.com/v1/check-groups
+    ```
+  </Tab>
+</Tabs>
+
+```ts Using Existing Group highlight={2,6}
+// Reference an existing group by ID
+const existingGroup = CheckGroupV2.fromId(123)
+
+new ApiCheck("existing-group-check", {
+  name: "Check with Existing Group",
+  group: existingGroup,
+  request: {
+    method: "GET",
+    url: "https://api.example.com/endpoint",
+  },
+})
+```
+
+When referencing existing groups with `fromId()`, note:
+- You cannot filter tests by group tags since group properties aren't available locally
+- Checks won't inherit the group's frequency. This only works for groups defined in the CLI project
+
+Consider creating a new group definition in your project if you need these features.
```

Comment:
I think it should be mentioned that you can import an existing group. Creating a new group is not your only option.

Reply:
You could for example run `npx checkly import check-group:123` to import just the group and nothing else. Or you can pick from the resource list yourself.

Reply:
Good point yeah I missed that, added now.